tests/testthat/test-utils.R

```diff
@@ -23,3 +23,19 @@ test_that("create_char_dataframe", {
   expect_equal(exp, got)
 
 })
+
+test_that("test random_string", {
+  set.seed(123)
+  suppressWarnings(rm(.openxlsx2.seed, envir = .GlobalEnv))
+
+  x <- .Random.seed
+  tmp <- openxlsx2:::random_string()
+  y <- .Random.seed
+  expect_identical(x, y)
+  expect_equal("HmPsw2WtYSxSgZ6t", tmp)
+
+  x <- .Random.seed
+  tmp <- openxlsx2:::random_string(length = 6)
```

Comment:
it is not helpfull to switch between branches back and forth, I have removed `openxlsx2:::` already